projects/go-lib/src/lib/components/go-header-bar/go-header-bar.component.scss

```diff
@@ -0,0 +1,39 @@
+@import "../../../styles/_variables.scss";
+.go-header-bar {
+  margin: -2rem;
+  height: 68px;
```

Comment:
We definitely do not want to assign a fixed height here. This header should be flexible based on the content within it.